src/Tizen.Applications.Common/Tizen.Applications.CoreBackend/EventType.cs

```diff
@@ -87,6 +87,12 @@ public class EventType
         /// <since_tizen> 3 </since_tizen>
         public static readonly EventType DeviceOrientationChanged = "DeviceOrientationChanged";
 
+        /// <summary>
+        /// Pre-defined event type "DeviceOrientationChanged".
```

Comment:
Do you better to add description Which EventArgs should be used. 
``` c#
AddEventHandler<SuspendedStateEventArgs>(EventType.SuspendedStateChanged, ...);
```
Because if not, it is not working

Reply:
Done